Subject: [PATCH] Fetch support for selinux in gnulib

This eliminates the need for the following patch:
findutils: support selinux and gnulib

This is done by fetching the latest gnulib updates
up to the point where the selinux-at module was
introduced from coreutils.

Upstream-Status: Backport

diff --git a/import-gnulib.config b/import-gnulib.config
index f2e8998..fa24d99 100644
--- a/import-gnulib.config
+++ b/import-gnulib.config
@@ -1,7 +1,8 @@
 # findutils gnulib.config -*- sh -*-
 
 # What version of gnulib to use?
-gnulib_version="e5573b1bad88bfabcda181b9e0125fb0c52b7d3b"
+gnulib_version="7dfa1250265017223e36c72713a7521421b4d27c"
 destdir="gnulib"
 
 # Random extra gnulib files needed for findutils.
@@ -68,6 +69,7 @@ realloc
 regex
 rpmatch
 savedir
+selinux-at
 stat-macros
 stat-time
 stpcpy
